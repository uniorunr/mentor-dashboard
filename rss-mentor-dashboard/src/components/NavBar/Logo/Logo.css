.navbar-logo {
  margin: 20px 0;
}

.navbar-logo a {
  color: black;
  text-decoration: none;
  font-weight: bold;
}