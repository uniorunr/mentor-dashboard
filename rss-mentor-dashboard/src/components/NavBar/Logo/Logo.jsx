import React from 'react';
import PropTypes from 'prop-types';
import './Logo.css';

const Logo = ({ link, brandName }) => (
  <h1 className="navbar-logo">
    <a href={link}>{brandName}</a>
  </h1>
);

Logo.propTypes = {
  link: PropTypes.string.isRequired,
  brandName: PropTypes.string.isRequired,
};

export default Logo;
