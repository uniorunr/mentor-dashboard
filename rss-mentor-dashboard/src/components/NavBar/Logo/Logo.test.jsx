import React from 'react';
import renderer from 'react-test-renderer';
import Logo from './Logo';

it('renders correctly', () => {
  const tree = renderer
    .create(<Logo link="test" brandName="test" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
