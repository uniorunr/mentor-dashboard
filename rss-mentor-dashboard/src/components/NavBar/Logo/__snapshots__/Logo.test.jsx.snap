// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h1
  className="navbar-logo"
>
  <a
    href="test"
  >
    test
  </a>
</h1>
`;
